src/stavenote.ts

```diff
@@ -1120,28 +1120,22 @@ export class StaveNote extends StemmableNote {
       const { yTop, yBottom } = this.getNoteHeadBounds();
       // eslint-disable-next-line
       const noteStemHeight = this.stem!.getHeight();
-      const flagX = this.getStemX();
+      const flagX = this.getStemX() - Tables.STEM_WIDTH / 2;
       // What's with the magic +/- 2
       // ANSWER: a corner of the note stem pokes out beyond the tip of the flag.
       // The extra +/- 2 pushes the flag glyph outward so it covers the stem entirely.
       // Alternatively, we could shorten the stem.
       const flagY =
         this.getStemDirection() === Stem.DOWN
           ? // Down stems are below the note head and have flags on the right.
-            yTop -
-            noteStemHeight +
-            2 -
-            (this.glyphProps ? this.glyphProps.stemDownExtension : 0) * this.getStaveNoteScale() -
-            (this.flag?.getMetrics().yShift ?? 0) * (1 - this.getStaveNoteScale())
+            yTop - noteStemHeight - this.flag.getTextMetrics().actualBoundingBoxDescent
```

Comment:
So these two lines of code (1131 and 1133) removed the magic constant of 2. Does that mean we no longer need it to fix the stem issue?

If so, you can remove the block comment above ("What's with the magic +/- 2.....").

Reply:
done